sourcerer-core/src/test/java/org/elder/sourcerer2/subscription/EventSubscriptionManagerTest.java

```diff
@@ -1,29 +1,32 @@
 package org.elder.sourcerer2.subscription;
 
 import com.google.common.collect.ImmutableMap;
+import org.elder.sourcerer2.EventId;
 import org.elder.sourcerer2.EventRecord;
 import org.elder.sourcerer2.EventRepository;
 import org.elder.sourcerer2.EventSubscriptionPositionSource;
 import org.elder.sourcerer2.EventSubscriptionUpdate;
+import org.elder.sourcerer2.RepositoryVersion;
+import org.elder.sourcerer2.StreamId;
+import org.elder.sourcerer2.StreamVersion;
 import org.elder.sourcerer2.SubscriptionToken;
 import org.elder.sourcerer2.SubscriptionWorkerConfig;
 import org.junit.Ignore;
 import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import reactor.core.publisher.Flux;
-import reactor.core.publisher.WorkQueueProcessor;
 
 import java.time.Instant;
 import java.util.List;
-import java.util.UUID;
 import java.util.stream.IntStream;
 
 import static org.mockito.Matchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class EventSubscriptionManagerTest {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add some tests to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.elder.org/coding_rules#rule_key=squid%3AS2187)